src/MOIWrapper.jl

```diff
@@ -374,3 +374,40 @@ end
 function LQOI.delete_variables!(instance::Optimizer, start_col::Int, end_col::Int)
     delete_columns(instance.inner, [Cint(i-1) for i in start_col:end_col])
 end
+
+const statmap = Dict(zip([ 0x00,  0x01, 0x02, 0x03, 0x04, 0x05],
```

Comment:
I'd say that `Dict(0x00 => MOI.BASIC, 0x01 => MOI.BASIC, ...)` is a bit better than the `zip` form because the mapping is easier to read.